ui/raidboss/data/03-hw/trial/thordan-ex.txt

```diff
@@ -0,0 +1,281 @@
+# but it's more important that we sync during the loop,
+# so we accept a small amount of jumpiness in the timeline.
+140.2 "--targetable--"
+150.2 "Divine Right" sync / 1[56]:[^:]*:(Ser Adelphel|Ser Janlenoux):1493:/
+159.3 "Holy Bladedance" sync / 1[56]:[^:]*:(Ser Adelphel|Ser Janlenoux):1496:/
+175.3 "Skyward Leap 1" sync / 1[56]:[^:]*:Ser Vellguine:14A9:/
+177.3 "Divine Right" sync / 1[56]:[^:]*:(Ser Adelphel|Ser Janlenoux):1493:/
+178.3 "Skyward Leap 2" sync / 1[56]:[^:]*:Ser Paulecrain:14A9:/
+179.4 "Heavenly Slash" sync / 1[56]:[^:]*:(Ser Adelphel|Ser Janlenoux):1494:/
+181.3 "Skyward Leap 3" sync / 1[56]:[^:]*:Ser Ignasse:14A9:/
+192.4 "Holiest Of Holy" sync / 1[56]:[^:]*:(Ser Adelphel|Ser Janlenoux):1495:/
+198.5 "Holy Bladedance" sync / 1[56]:[^:]*:(Ser Adelphel|Ser Janlenoux):1496:/
+205.6 "Divine Right" sync / 1[56]:[^:]*:(Ser Adelphel|Ser Janlenoux):1493:/
+212.7 "Holiest Of Holy" sync / 1[56]:[^:]*:(Ser Adelphel|Ser Janlenoux):1495:/
+216.8 "Heavenly Slash" sync / 1[56]:[^:]*:(Ser Adelphel|Ser Janlenoux):1494:/
+
+232.9 "Divine Right" sync / 1[56]:[^:]*:(Ser Adelphel|Ser Janlenoux):1493:/
+242.0 "Holy Bladedance" sync / 1[56]:[^:]*:(Ser Adelphel|Ser Janlenoux):1496:/
+260.0 "Divine Right" sync / 1[56]:[^:]*:(Ser Adelphel|Ser Janlenoux):1493:/
+262.1 "Heavenly Slash" sync / 1[56]:[^:]*:(Ser Adelphel|Ser Janlenoux):1494:/
+275.2 "Holiest Of Holy" sync / 1[56]:[^:]*:(Ser Adelphel|Ser Janlenoux):1495:/
+281.3 "Holy Bladedance" sync / 1[56]:[^:]*:(Ser Adelphel|Ser Janlenoux):1496:/
+288.3 "Divine Right" sync / 1[56]:[^:]*:(Ser Adelphel|Ser Janlenoux):1493:/
+295.4 "Holiest Of Holy" sync / 1[56]:[^:]*:(Ser Adelphel|Ser Janlenoux):1495:/
+299.5 "Heavenly Slash" sync / 1[56]:[^:]*:(Ser Adelphel|Ser Janlenoux):1494:/ forcejump 216.8
+
+
+# Intermission part 3
+350.0 "--sync--" sync / 1[56]:[^:]*:Ser Paulecrain:1018:/ window 150,10
+352.0 "--sync--" sync / 1[56]:[^:]*:Ser Grinnaux:1018:/ window 150,10
+353.0 "--sync--" sync / 1[56]:[^:]*:Ser Haumeric:1018:/ window 150,10
+359.6 "Hiemal Storm" sync / 1[56]:[^:]*:Ser Haumeric:14AE:/ window 328,10
+359.8 "--sync--" sync / 1[56]:[^:]*:Ser Noudenet:14AF:/ # Hiemal Storm
+360.2 "Spiral Pierce" sync / 1[56]:[^:]*:(Ser Ignasse|Ser Paulecrain|Ser Vellguine):14A7:/
+361.5 "Dimensional Collapse" sync / 1[56]:[^:]*:Ser Grinnaux:1499:/
+361.9 "--sync--" sync / 1[56]:[^:]*:Ser Grinnaux:149A:/ # Dimensional Collapse
+365.0 "Faith Unmoving" sync / 1[56]:[^:]*:Ser Grinnaux:149B:/
+
+# Intermission part 4
+# Two sets of four Heavy Impacts used during this intermission.
+# They overlap heavily. Because they use different IDs for each ring,
+# it's safe to leave them all synced.
+366.0 "Holy Meteor" sync / 1[56]:[^:]*:Ser Noudenet:14B0:/ window 220,10
+374.6 "--sync--" sync / 1[56]:[^:]*:Ser Guerrique:149F:/ # Heavy Impact (intial cast, no earth ring)
+376.0 "Comet x4" duration 4 #sync / 1[56]:[^:]*:Ser Noudenet:14B5:/
+379.4 "Heavy Impact (round 1 ring 1)" sync / 1[56]:[^:]*:Ser Guerrique:14A0:/
```

Comment:
this is a little long; I think it could just say ring 1-4 and then again ring 1-4 (even if interleaved)

(alternatively, it could be a duration)

Reply:
Tried the duration, it seems to be okayish. People can complain at release if it isn't great.